pom.xml

```diff
@@ -56,9 +57,11 @@
 			<url>https://repository.apache.org/content/repositories/snapshots/</url>
 			<releases>
 				<enabled>false</enabled>
+				<updatePolicy>never</updatePolicy>
 			</releases>
 			<snapshots>
-				<enabled>true</enabled>
+				<enabled>false</enabled>
+				<updatePolicy>never</updatePolicy>
```

Comment:
Right! I got my build into a weird half-maven half-local state so switched off snapshots to debug. It was caused by the *_2.10 suffixing change not being fully propagated to these deps. Normal users should never have that problem.